components/organisms/TopicBox.js

```diff
@@ -13,22 +13,25 @@ export default function TopicBox(props) {
         <div className="col-span-2">
           <h3 className="pt-2 pb-2 font-extrabold">{props.title}</h3>
         </div>
-        <div className="col-span-1 pt-2">
+        <div className="col-span-1">
           <Image
             src={props.image}
             alt={props.imgalt}
             width={73}
             height={73}
+            layout="responsive"
           ></Image>
         </div>
       </div>
-      <p>{props.title}</p>
-      <br />
-      <p className="font-bold">Find information about</p>
-      <p>thing 1</p>
-      <p>thing 2</p>
-      <p>thing 3</p>
+      <h4>{props.title}</h4>
+      <p>{props.body}</p>
       <br />
+      <h4 className="font-bold pb-4">Find information about</h4>
```

Comment:
The top title should be bigger than the bottom one, so maybe an `h4` and an `h5`?

Reply:
Also, shouldn't headers be bold by default?

Reply:
ya, good call, lingered from before i made it heads